cruise-control/src/main/java/com/linkedin/kafka/cruisecontrol/servlet/UserTaskManager.java

```diff
@@ -408,11 +408,11 @@ public synchronized UserTaskInfo markTaskInExecution(String uuid) {
   }
 
   /**
-   * Unmark the in-execution user task once {@link com.linkedin.kafka.cruisecontrol.executor.Executor} has finish the execution.
+   * Update user task status once {@link com.linkedin.kafka.cruisecontrol.executor.Executor} has finished the execution.
    *
    * @param uuid UUID associated with the in-execution user task.
    */
-  public synchronized void markTaskFinishExecution(String uuid) {
+  public synchronized void taskExecutionFinished(String uuid) {
```

Comment:
Nit: The name change `markTaskFinishExecution` -> `taskExecutionFinished` makes the functionality of the method not clear (e.g. is it a getter or setter)?